code/modules/antagonists/changeling/changeling.dm

```diff
@@ -394,16 +394,38 @@
 		to_chat(owner.current, span_warning("We cannot evolve this ability!"))
 		return FALSE
 
+	//Pre-requisite power check
+	var/datum/action/changeling/power = new sting_path()
+	if(!can_give(power))
+		to_chat(owner.current, span_warning("We lack the pre-requisite abilities to evolve this ability"))
+		return FALSE
+	qdel(power)
```

Comment:
Rather than qdel it on success, I was imagining we qdel it on failure and instead pass it into `gain_power` to be used there

Cause now we're newing the power twice, kinda wasteful.